src/nupic/engine/Region.cpp

```diff
@@ -61,13 +58,15 @@ Region::Region(std::string name, const std::string &nodeType,
   RegionImplFactory &factory = RegionImplFactory::getInstance();
   spec_ = factory.getSpec(nodeType);
 
+  // Dimensions start off as don't care but can be changed by user. dek 02/2019
+  // singleNodeOnly is ignored.
+  //dims_.push_back(1);
 
   // Dimensions start off as unspecified, but if
   // the RegionImpl only supports a single node, we
   // can immediately set the dimensions.
   if (spec_->singleNodeOnly)
     dims_.push_back(1);
-  // else dims_ = []
```

Comment:
if yes, this IF could be removed/commented out?

Reply:
You are correct. If I add this feature then the singleNodeOnly flag has no meaning.  The default would be we would always set the dimension to 1 which means single dimension with width determined by region parameters.

Reply:
I will fix that.